packaging/linux/flatpak/dev.lizardbyte.app.Sunshine.yml

```diff
@@ -49,23 +45,25 @@ modules:
   - "modules/cuda.json"
 
   - name: sunshine
-    disabled: false
-    buildsystem: cmake-ninja
-    no-make-install: false
     builddir: true
+    build-commands:
+      - npm install --offline
```

Comment:
This doesn't run before it fails, try with a new cmake option to make npm install offline.